.github/issue_template.md

```diff
@@ -0,0 +1,13 @@
+## Checklist
+- [ ] I have searched the existing issues and pull requests.
```

Comment:
I would just modify this to say, "I have searched the existing issues and pull requests for duplicates."